docs/source/kafka_check.rst

```diff
@@ -35,3 +35,37 @@ In case of min isr violations:
     CRITICAL: 1 partition(s) have the number of replicas in sync that is lower
     than the specified min ISR.
 
+Checking under replicated partitions
+====================================
+This kafka tool provides the ability to check number of under replicated partitions for all
```

Comment:
Would be 'ability to check' or 'ability to report' or 'ability to check and report'?